doc/install.rst

```diff
@@ -21,6 +21,9 @@ Scikit-learn requires:
 - NumPy (>= 1.11.0),
 - SciPy (>= 0.17.0).
 
+Scikit-learn plotting capabilities require Matplotlib (>= 1.5.1). Some of the
+scikit-learn examples might require one or more extra dependencies:
+scikit-image (>= 0.12.3), pandas (>= 0.18.0), and joblib (>= 0.11)
```

Comment:
joblib is not an extra dependency -- you won't be able to `import sklearn` without it ;)

Reply:
true from now on.